pept/tracking/birmingham_method/birmingham_method.py

```diff
@@ -0,0 +1,231 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import pept
+import os
+import matplotlib.lines as mlines
+
+from .extensions.birmingham_method import birmingham_method
+
+class BirminghamMethod():
+	'''
+
+	The Birmingham Method is an analytical technique for tracking particles using PEPT data from LORs.
+
+	'''
+
+	def __init__(self):
+		print("Initialised BirminghamMethod")
+		self._line_data = None
+
+		
+	@property
+	def line_data(self):
```

Comment:
We can just supply line_data to the tracking methods

Reply:
Done!